src/bootstrap.php

```diff
@@ -182,7 +185,10 @@
                 ->forType('correction', 'editorial', 'feature', 'insight', 'research-advance', 'research-article', 'retraction', 'registered-report', 'replication-study', 'scientific-correspondence', 'short-report', 'tools-resources')
                 ->sortBy('date')
                 ->forSubject($subject->getId())
-                ->slice(0, 5);
+                ->slice(0, 5)
+                ->filter(function (Article $article) use ($id) {
```

Comment:
callback could probably be extracted into `$doesNotRecommendItself`

Reply:
👍